daemon/daemon.go

```diff
@@ -1056,7 +1058,10 @@ func writeDistributionProgress(cancelFunc func(), outStream io.Writer, progressC
 
 	for prog := range progressChan {
 		if err := progressOutput.WriteProgress(prog); err != nil && !operationCancelled {
-			logrus.Errorf("error writing progress to client: %v", err)
+			// don't log broken pipe errors as this is the normal case when a client aborts
+			if !isBrokenPipe(err) {
+				logrus.Errorf("error writing progress to client: %v", err)
```

Comment:
Ok, adding this, but also note it is not currently possible to determine what that session was actually pulling, so the message is a bit naked.